internal/dataplane/sendconfig/inmemory.go

```diff
@@ -57,7 +57,8 @@ func (s UpdateStrategyInMemory) Update(ctx context.Context, targetState ContentW
 	}
 
 	flattenErrors := shouldUseFlattenedErrors()
-	if errBody, err := s.configService.ReloadDeclarativeRawConfig(ctx, bytes.NewReader(config), true, flattenErrors); err != nil {
+	errBody, err := s.configService.ReloadDeclarativeRawConfig(ctx, bytes.NewReader(config), true, flattenErrors)
```

Comment:
When `err` is not `nil` the other returned value in this case `errBody` is treated as valid and not something to discard, it's not idiomatic Go to do like that

Reply:
Yeah, I agree that `ReloadDeclarativeRawConfig` could have a better interface and return just a single error, allowing inspection of the error body by casting the error to some predefined custom error struct or something similar.

But in this case, I think we'll have to ignore it as `ReloadDeclarativeRawConfig` is not defined in KIC but `go-kong` and it cannot be improved in this PR.